builds/jenkins-build-pack-sls-app/Jenkinsfile

```diff
@@ -398,6 +396,31 @@ node() {
   }
 }
 
+def attachTagstoLogGroup(config, environment_logical_id, credsId){
+  try {
+    def deploymentDescriptor = null
+    if( env_deployment_descriptor != null ){
+      deploymentDescriptor = env_deployment_descriptor
+    } else {
+      deploymentDescriptor = config['deployment_descriptor']
+    }
+    // If no descriptor present then simply making an empty one. The readYaml default behavior is to return empty string back that is harmful as Map not String is expected below
+    def doc = deploymentDescriptor ? readYaml(text: deploymentDescriptor) : [:]
+    def functionsElem = doc['functions']
+
+    // changing variable names appropriately
+    def serviceDomain = "${configLoader.INSTANCE_PREFIX}-${config['domain']}-${config['service']}"
+    functionsElem.each{functionName, funcConfig ->
+      def lambdaName = funcConfig['name'] ? funcConfig['name'] : "${serviceDomain}-${functionName}-${environment_logical_id}"
+      def logGroupName = "/aws/lambda/${lambdaName}"
+      def serviceName = config['service']
+      def namespace = config['domain']
+      def environment = environment_logical_id
+      sh "aws logs tag-log-group --log-group-name ${logGroupName} --tags '{\"service\":\"${serviceName}\", \"namespace\":\"${namespace}\", \"environment\":\"${environment}\", \"functionName\":\"${lambdaName}\"}' --profile ${credsId}"
```

Comment:
Did we check for any plugins that can do this via serverless.yml? It would be great if we can accomplish this through serverless framework (and reduce our code footprint).

Reply:
https://github.com/gfragoso/serverless-tag-cloud-watch-logs